package.json

```diff
@@ -9,17 +9,19 @@
     }
   },
   "dependencies": {
+    "lodash": "~2.4.1",
+    "q": "~0.9.7",
+    "vumigo_v01": "~0.1.22",
+    "vumigo_v02": "~0.2"
+  },
+  "optionalDependencies": {
```

Comment:
Ya, ideally those deps shouldn't be in the vumi-go repo, they're needed in production, not here. A better solution would be to have a separate `package.json` for production with these dependencies in that doesn't live in this repo. Maybe we should do something like that instead? In the mean time, I do think its going to be safer to not make these deps optional here. There's potential for deploys to go bad if dependencies that need to be deployed aren't deployed.